modules/reviewing-cluster-status-from-the-openshift-cluster-manager.adoc

```diff
@@ -35,18 +30,21 @@ Alternatively, you can navigate to {cluster-manager-url} directly and select you
 +
 * Subscription information, including the service level agreement (SLA) status, the subscription unit type, the production status of the cluster, the subscription obligation, and the service level
 +
-* A cluster history
+[TIP]
+====
+To view the history for your cluster, click the *Cluster history* tab.
+====
 
-. Navigate to the *Observe* page to review the following information:
+. Navigate to the *Monitoring* page to review the following information:
 * A list of any issues that have been detected
 +
 * A list of alerts that are firing
 +
 * The cluster Operator status and version
 +
-* Cluster resource usage
+* The cluster's resource usage
 
-. Navigate to the *Insights* page to review the following information provided by Red Hat Insights:
+. _Optional:_ You can view information about your cluster that Red Hat Insights collects by navigating to the *Overview* menu. From this menu you can view the following information:
```

Comment:
Suggest adding comma after introductory phrase per IBM Style.
```suggestion
. _Optional:_ You can view information about your cluster that Red Hat Insights collects by navigating to the *Overview* menu. From this menu, you can view the following information:
```

Reply:
I don't think "Optional" is italicized normally, but I can't find the specific guidance in the style guides.